packages/dd-trace/src/config.js

```diff
@@ -714,8 +714,8 @@ class Config {
     const env = setHiddenProperty(this, '_env', {})
     setHiddenProperty(this, '_envUnprocessed', {})
 
-    tagger.add(tags, OTEL_RESOURCE_ATTRIBUTES, true)
-    tagger.add(tags, DD_TAGS)
+    parseSpaceSeparatedTags(tags, handleOtel(OTEL_RESOURCE_ATTRIBUTES))
+    parseSpaceSeparatedTags(tags, DD_TAGS)
```

Comment:
Nit: after looking at it again, we could change `parseSpaceSeparatedTags` to only change the format and move `tagger.add()` out again. That way the method only does a single thing which is always good :)